src/helpers.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Symfony\Component\Process\Process;
+
+if (! function_exists('create_hooks_dir')) {
+    /**
+     * Create hook directory if not exists.
+     *
+     * @param  string  $dir
+     * @param  int  $mode
+     * @param  bool  $recursive
+     *
+     * @return void
+     */
+    function create_hooks_dir($dir, $mode = 0700, $recursive = true)
+    {
+        if (! is_dir("{$dir}/hooks")) {
+            mkdir("{$dir}/hooks", $mode, $recursive);
+        }
+    }
+}
+
+if (! function_exists('git_version')) {
+    /**
+     * Get latest git tag version.
+     *
+     * @return string
+     */
+    function git_version()
+    {
+        $process = new Process('git describe --tags $(git rev-list --tags --max-count=1)');
```

Comment:
This now requires [symfony/process](https://github.com/symfony/process) as a project dependancy, right?